types/vscode/package.json

```diff
@@ -2,6 +2,8 @@
     "private": true,
     "name": "@types/vscode",
     "version": "1.106.9999",
+    "nonNpm": "conflict",
+    "nonNpmDescription": "TypeScript definitions for the Visual Studio Code Extension API",
```

Comment:
@mjbvz Is this file also generated by you? If so, I'd like to make sure these two lines are generated too.

Reply:
Note to self https://github.com/microsoft/vscode/tree/main/build/azure-pipelines/publish-types